src/documents/ModelRelay.md

```diff
@@ -187,22 +226,38 @@ the updates will propagate only in one direction. This can still be highly usefu
 
 #### Controlling propagation through a relay rule ####
 
-Each explicit relay rule can accept options `forward` and `backward` which allows the configurer to control the occasions on which the relay is operated in those directions &#8212; that is, `forward` representing
-the direction from `source` to `target`, and `backward` representing the direction from `target` to `source`. If the transform is not
-invertible, the `backward` option is ignored, and the system will behave as if it read `backward: "never"`. The control which can be exerted is over whether the relay is operated either during the
-<a href="#the-initial-transaction">initial transaction</a>, the "mature life" of the component after construction, or neither, or both. The default value for these options is `"always"` which
-implies that the system will always operate the relay rule if it can. 
+Each explicit relay rule can accept options `forward` and `backward` which allows the configurer to 
+control the occasions on which the relay is operated in those directions &#8212; that is, `forward` 
+representing the direction from `source` to `target`, and `backward` representing the direction 
+from `target` to `source`. If the transform is not invertible, the `backward` option is ignored, 
+and the system will behave as if it read `backward: {excludeSource: "*"}`.
+ 
+Some use cases for controlling propagation relative to the `init` source are: 
```

Comment:
Good idea - I've added a (working!) sample. wrt the model listener examples - this has only become "the same concept" as a result of this rewrite - it's also not precisely the same since a source value of "local" is meaningless for a model relay rule.